roles/pulp_repos/README.md

```diff
@@ -0,0 +1,23 @@
+pulp_repos
+==========
+
+Role to enable repositories needed to install pulp. It can be disabled if user already have all the repositories enabled.
```

Comment:
If it's meant to be called by import_role / include_role or as a dependency, we would mention that here.